lib/OrganizationsFieldArray/OrganizationCard.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Card,
+  IconButton,
+} from '@folio/stripes/components';
+import OrgInfo from './OrgInfo';
+
+export default class OrganizationCard extends React.Component {
```

Comment:
It might make sense to call this something like `EditOrganizationCard`, since we're gonna have a card within it that should be the one that's _actually_ called `OrganizationCard`.